release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
 
 |====
 
+[id="ocp-4-5-4-ibm-power"]
+===== IBM Power Systems
+
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power] or xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power in a restricted network].
+
+[discrete]
+====== Restrictions
+
+Note the following restrictions for {product-title} on IBM Power:
+
+* {product-title} for IBM Power Systems does not include the following Technology Preview features:
++
+** Container-native virtualization (CNV)
```

Comment:
CNV has GA'd with the name "OpenShift Virtualization"

@vikram-redhat, will you double-check this list?